README.md

```diff
@@ -3,9 +3,33 @@
 This framework is not yet stable and is subject to change, but is available
 for early testing.
 
-## Getting Started
+# Getting Started
 
-Your charm directory should have the following overall structure:
+## Table of contents
+
+[Introduction](##Introduction)
+
+[Charm Creation](##Charm%20Creation)
+
+## Introduction
+
+### Juju and Charms
+
+The Charm Operator Framework is the companion to [Juju](https://jaas.ai/docs/what-is-juju) charms are the workers, and juju is the orchestrator, together they encapsulate the complexity of Day zero through to day three operations away from the user making infrastructure as a code a reality.
+
+The charm operator framework was created to answer this not just for VM/Baremetal scenario but for containers also.
+
+Here is a diagram of how juju and charms work together:
+
+![Juju and charms](./diagrams/juju_charms.png)
+
+## Charm Creation Quick Start
+
+There are currently two types of Charm, IAAS charms and Kubernetes (K8s) charms.
```

Comment:
Adding that statement isn't really a "quickstart". We should take an opinionated view once we've indicated that people are going to work through a tutorial. Perhaps move to the introduction?

Reply:
Moved